java-action-archetype/README.md

```diff
@@ -0,0 +1,14 @@
+# Maven Archetype for Java Action
+
+This archetype helps to generate the JavaAction template project
+
+## Generate project 
+
+```sh
+mvn archetype:generate \
+  -DarchetypeGroupId=org.apache.openwhisk.java \
+  -DarchetypeArtifactId=java-action-archetype \
+  -DarchetypeVersion=1.0-SNAPSHOT \
+  -DgroupId=com.example \
+  -DartifactId=demo-function
+```
```

Comment:
Can we also continue the readme with what to do afterwards, in order to test the action ? _explaining like I'm five_ 😄

Reply:
updated